go.mod

```diff
@@ -44,3 +44,8 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
+
+// NOTE: Temporary for testing purposes
+replace github.com/gophercloud/gophercloud/v2 => github.com/Sharpz7/gophercloud/v2 v2.0.0-20240719114351-6815cc5264b6
+
+replace github.com/gophercloud/gophercloud => github.com/Sharpz7/gophercloud v0.0.0-20240719142242-07fde05c6650
```

Comment:
This was rejected as a backport for v1

https://github.com/gophercloud/gophercloud/pull/3137.

I will instead migrate the relavent code to v2. See https://github.com/openstack-exporter/openstack-exporter/issues/385